requirements-test.txt

```diff
@@ -30,4 +30,6 @@ ansible-core<2.16.3
 molecule>=5.0.1
 molecule-plugins[docker]
 pre-commit
-pytest-testinfra
+# pytest-testinfra contains a fix for SystemdService.exists that is
+# required by this role's Molecule test code.
+pytest-testinfra>=10.1.1
```

Comment:
This should be moved into [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) (along with a dependabot ignore statement).

Reply:
Please see cisagov/skeleton-ansible-role#201.